tests/ui/documents/test_documents_views.py

```
@@ -115,6 +117,27 @@ def test_store_organisation(client, db, organisation):
     assert not g.get("organisation")
 
 
+def test_global_view_blocked_on_dedicated_portal(client, organisation):
```

Ruff (0.15.2)

[warning] Unused function argument: `organisation`

(ARG001)